src/main/java/com/company/Main.java

```diff
@@ -1,32 +1,97 @@
-        else if (operator.equalsIgnoreCase("div") || operator.equals("/"))
-            result = value1 / value2;
-        else System.out.println("Wrong value, pls try again");
-
-        System.out.println("Your result  " + String.format("%.2f%n", result));
+        System.out.println("Pls enter through space parameters for first Array: ");
+        System.out.println("initial value, size of array, step of array, increment operator((+),(*),(-),(^) or (/))");
+        var arrayParametersForFirstArray = in.nextLine();
+        System.out.println("Pls enter through space parameters for second Array: ");
+        System.out.println("initial value, size of array, step of array, increment operator((+),(*),(-),(^) or (/))");
+        var arrayParametersForSecondArray = in.nextLine();
+        System.out.println("Pls enter operator to calculate the third array (+),(*),(-),(^) or (/)");
+        var generalOperator = in.nextLine();
 
+
+        String[] parametersForFirstArray = arrayParametersForFirstArray.split(" ");
+        String[] parametersForSecondArray = arrayParametersForSecondArray.split(" ");
+
+        validateArrayLength(parametersForFirstArray);
+        validateArrayLength(parametersForSecondArray);
+
+        final var initialValueOfFirstArray = Double.parseDouble(parametersForFirstArray[0]);
+        final var initialValueOfSecondArray = Double.parseDouble(parametersForSecondArray[0]);
+
+        final var sizeOfFirstArray = Integer.parseInt(parametersForFirstArray[1]);
+        final var sizeOfSecondArray = Integer.parseInt(parametersForSecondArray[1]);
+
+        final var stepOfFirstArray = Double.parseDouble(parametersForFirstArray[2]);
+        final var stepOfSecondArray = Double.parseDouble(parametersForSecondArray[2]);
+
+        String operatorOfFirstArray = parametersForFirstArray[3];
+        String operatorOfSecondArray = parametersForSecondArray[3];
+
+        double[] firstArray = createArray(initialValueOfFirstArray, sizeOfFirstArray, stepOfFirstArray, operatorOfFirstArray);
+        double[] secondArray = createArray(initialValueOfSecondArray, sizeOfSecondArray, stepOfSecondArray, operatorOfSecondArray);
+
+        System.out.println("First Array" + Arrays.toString(firstArray));
+        System.out.println("Second array" + Arrays.toString(secondArray));
+
+        if (sizeOfFirstArray == sizeOfSecondArray) {
+
+            System.out.println("The result of " + generalOperator + " entered arrays is " + Arrays.toString(calculateArrays(firstArray, secondArray, generalOperator)));
+
+        } else
+            System.out.println("Please enter the same size of arrays");
     }
 
+    public static double[] createArray(double initialValue, int sizeOfArray, double stepOfArray, String incrementOperator) {
+        double[] array = new double[sizeOfArray];
+        array[0] = initialValue;
+        for (int i = 1; i < array.length; i++) {
+            array[i] = calculateNumber(initialValue, stepOfArray, incrementOperator);
```

Comment:
With this method call, you will always have the same value in your array. 
For each element of the array, you are just performing operations on the initial value again and again.